models/models.js

```diff
@@ -67,7 +67,15 @@ exports.fetchArticles = () => {
     });
 };
 
-exports.fetchComments = (articleId) => {
+exports.fetchComments = async (articleId) => {
+  await db
+    .query("SELECT * FROM articles WHERE article_Id=$1", [articleId])
+    .then(({ rows }) => {
+      if (rows.length === 0) {
+        return Promise.reject({ status: 404, msg: "No article found" });
+      }
+    });
+
```

Comment:
Don't mix async/await with promises inside the same function & ideally not inside the same project!